tests/network/hotplug_bridge.go

```diff
@@ -382,18 +382,21 @@ func removeInterface(vm *v1.VirtualMachine, name string) error {
 func verifyBridgeDynamicInterfaceChange(vmi *v1.VirtualMachineInstance, plugMethod hotplugMethod) *v1.VirtualMachineInstance {
 	virtClient := kubevirt.Client()
 	if plugMethod == migrationBased {
+		postOperationTimestamp := time.Now()
```

Comment:
The assumption that the client on which this test runs is in total sync with the API server time is problematic.
Comparisons like these should be done based on one clock.

Reply:
while true, since we have enough grace i think it shouldn't block us
there are potentially other possible races as well, but we can also live with them if we take the VMIM approach
more info here https://github.com/kubevirt/kubevirt/pull/15209#discussion_r2212454142

(btw what about using UID of the previous VMIM, would it help ?)